spec/queryrenderer/AggregateFunctionAppender_spec.lua

```diff
@@ -0,0 +1,46 @@
+package.path = "src/?.lua;" .. package.path
+require("busted.runner")()
+local literal = require("queryrenderer.literal_constructors")
+local Query = require("exasolvs.Query")
+local AggregateFunctionAppender = require("exasolvs.queryrenderer.AggregateFunctionAppender")
+
+function it_asserts(expected, actual, explanation)
```

Comment:
Global function by intention?
```suggestion
local function it_asserts(expected, actual, explanation)
```